src/CMakeLists.txt

```diff
@@ -283,10 +283,13 @@ set(requires_private_for_pc_file "${requires_private_string}" PARENT_SCOPE)
 
 write_basic_package_version_file(exiv2ConfigVersion.cmake COMPATIBILITY ExactVersion)
 
-install(TARGETS exiv2lib EXPORT exiv2Config
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+install(TARGETS exiv2lib EXPORT exiv2Export)
```

Comment:
In CMake 3.14 and above, you don't need to supply the RUNTIME, LIBRARY, and ARCHIVE directories. This relies on the defaults, which conveniently were the same as before.

Reply:
Yup! This cmake version required to `build` exiv2 is `3.16.3` which is when this line of code runs. 

The version required to `consume` exiv2 is now `3.5`.

They are independent.